new_state/main.tf

```diff
@@ -5,7 +5,7 @@ terraform {
       version = ">= 3.24.1"
     }
   }
-  required_version = "~> 0.14"
+  required_version = ">= 0.14"
```

Comment:
```suggestion
  required_version = ">= 0.15.2"
```

Reply:
Hi! Thanks for opening this. Update to the version constraint here and then we'll merge.